config/fields/mixins/picker.php

```diff
@@ -68,13 +69,13 @@
 		},
 
 		/**
-		 * Whether to store UUID or path in
-		 * the content file
+		 * Whether to store UUID in the content file
+		 * or ID of the model
```

Comment:
```suggestion
		 * Whether to store UUID or ID in the 
		 * content file of the model
```

Reply:
Thanks for reply. I got it what you mean 👍